src/System.Net.Security/tests/FunctionalTests/ClientAsyncAuthenticateTest.cs

```diff
@@ -24,6 +24,17 @@ public ClientAsyncAuthenticateTest()
             _log = TestLogging.GetInstance();
         }
 
+        [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "CI uses old Framework version, which doesn't support using SslProtocols.None for SystemDefaultTlsVersions behavior")]
```

Comment:
@caesar1995, what version of netfx is required?  Not blocking this PR, but we should a) try to get CI updated to the latest version as soon as possible, and b) track re-enabling these tests when an appropriate version is in play.
cc: @dotnet/dnceng

Reply:
We need at least .NET Framework 4.7 for using SslProtocols.None as SystemDefaultTlsVersions behavior.